src/vivarium_public_health/metrics/disability.py

```diff
@@ -116,3 +113,36 @@ def get_disability_weight_pipeline(self, builder: Builder) -> Pipeline:
 
     def disability_weight_aggregator(self, dw: pd.DataFrame) -> float:
         return (dw * to_years(self.step_size)).sum().squeeze()
+
+    ##################
+    # Report methods #
+    ##################
+
+    def report(
+        self,
+        measure: str,
+        results: pd.DataFrame,
+    ) -> None:
+        """Combine the measure-specific observer results and save to a single file."""
+        results_dir = Path(self.results_dir)
+        measure, cause = measure.split("due_to")
+        measure = measure.strip("_")
+        cause = cause.strip("_")
+        # Add extra cols
+        results["measure"] = measure
+        results["cause"] = cause
+        results["random_seed"] = self.random_seed
+        results["input_draw"] = self.input_draw
+        # Sort the columns such that the stratifications (index) are first
+        # and "value" is last and sort the rows by the stratifications.
+        other_cols = [c for c in results.columns if c != "value"]
+        results = results[other_cols + ["value"]].sort_index().reset_index()
+
+        # Concat and save
+        results_file = results_dir / f"{measure}.csv"
```

Comment:
Is it possible that different observers could have the same measure (and therefore try to write to the same file?

Reply:
Hmm, so the measure is actually just the observation name so I guess if you registered more than one observer w/ the same name that would be a problem. I'll create a ticket to check for duplicate names and raise if so

Reply:
But that nuance aside, yes this could be a problem. Is it a realistic thing though to have different observers observing the same thing? I'm not sure how offhand to check for this.